### ISSUE TITLE
Fix apis.guru openapi 3.0 yaml

### ISSUE THREAD

**Issue by Jolanrensen** [APIs-guru/openapi-directory#1068]
Your own openapi spec appears out of date:
`ApiVersion.updated` seems to be non required anymore.
`Metrics` is missing almost all attributes while having `additionalProperties: false`

**Comment by MikeRalphson** [comment]
Well that's embarrassing! Thanks for pointing these issues out. I've temporarily changed `Metrics` to allow `additionalProperties` and hopefully fixed the missing `ApiVersion.updated` dates (won't show up in the API until tomorrow).

**Comment by MikeRalphson** [comment]
Thanks. This is related to #542 - I'd like to keep the API definition as-is and work on fixing the incorrect data, but is it causing you a downstream problem?

**Comment by Jolanrensen** [comment]
Yes, it breaks in the [OpenApi support of Kotlin Dataframe](https://kotlin.github.io/dataframe/gradle.html#openapi-schemas), since it tries to read the `info` object, which is required according to the spec, and it isn't present. 
But most likely, this implementation isn't the only thing that would break if received data isn't up to spec.

The specific bug happens at:
azure.com:azsadmin-quotas, 
azure.com:sql-databaseSecurityAlertPolicies, 
azure.com:sql-serverAzureADAdministrators
